Python-Basics.md

```diff
@@ -0,0 +1,710 @@
+# Python Basics
+
+## Objects
+
+> In Python, everything is an *object*.
+
+*Objects* represent a logical grouping of attributes. Attributes are data and/or functions. When an object is created in Python it is created with an *identity*, *type*, and *value*.
+
+In other languages, *primitives* are *values* that have no *properties* (attributes). For example, in javascript `undefined`, `null`, `boolean`, `string`, `number`, and `symbol` (new in ECMAScript 2015) are primitives.
+
+In Python, there are no primitives. `None`, *booleans*, *strings*, *numbers*, and even *functions* are all *objects* regardless how they are created.
+
+We can demonstrate this using some built-in functions:
+
+* `id`:  https://docs.python.org/3/library/functions.html#id
```

Comment:
[`id`](https://docs.python.org/3/library/functions.html#id)

You could use that format for these. I think is always better to hide the links like this and looks better.